awx/main/models/unified_jobs.py

```diff
@@ -492,6 +492,18 @@ def model_to_str(self):
         return UnifiedJobTypeStringMixin._camel_to_underscore(self.__class__.__name__)
 
 
+class UnifiedJobDeprecatedStdout(models.Model):
+
+    class Meta:
+        managed = False
```

Comment:
The key to this change is here:

1.  Remove `result_stdout_text` from `UnifiedJob` so it's no longer a field that gets tacked onto every query that the ORM builds.
2.  Introduce a migration that allows the field to be removed _without_ dropping the underlying column: https://github.com/ansible/awx/pull/795/files#diff-0d54c0f6b7264d1e5a102d50f1417810R23
3.  Add the `result_stdout_text` field to this _new_ model, which is marked as `managed = False` and `db_table = 'main_unifiedjob'`.  This means that this model will pull its field data from the `main_unifiedjob` instead of creating a new table for this model; this approach is what you do when you want to manage database views in Django, and if you think about this, it's sort of the same thing.

Reply:
👍